org.eclipse.xtext.builder/build.properties

```diff
@@ -2,6 +2,7 @@ source.. = src/,\
            emf-gen/,\
            xtend-gen/,\
            deprecated/
+output.. = target/classes/
```

Comment:
I think this should be "bin" ... otherwise it will probably clash between maven and eclipse build.

Reply:
So we need to change the classpath / project configuration? Right now it compiles to target/classes in eclipse.

Reply:
do we have any concept for that?
are the tycho recommendations for that?
separate/same output folder for eclipse and build?

Reply:
To my knowledge it is best to seperate eclipse and maven output folders. Otherwise mamven cleans eclipse class files and IDE is not so happy.

Reply:
hmm the current config is rather mixed

Reply:
I'm on it

Reply:
Done